edgecontext.thrift

```diff
@@ -16,6 +16,11 @@ e.g. en, en_US
 */
 typedef string LocaleCode
 
+/** Locale code forced to the BCP-47 format (e.g. de-DE, pt-BR, etc.).
```

Comment:
🔕 I feel that the more commonly used format is `_` instead of `-` (e.g. `pt_BR` instead of `pt-BR`), but as long as there's a standard for it I'm fine with it.